rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_participant_info.hpp

```diff
@@ -46,16 +54,16 @@
 #include "rmw_fastrtps_shared_cpp/custom_event_info.hpp"
 #include "rmw_fastrtps_shared_cpp/qos.hpp"
 #include "rmw_fastrtps_shared_cpp/rmw_common.hpp"
+#include "rmw_fastrtps_shared_cpp/utils/net_utils.hpp"
 
 using rmw_dds_common::operator<<;
 
 class ParticipantListener;
 
-enum class publishing_mode_t
-{
-  ASYNCHRONOUS,  // Asynchronous publishing mode
-  SYNCHRONOUS,   // Synchronous publishing mode
-  AUTO           // Use publishing mode set in XML file or Fast DDS default
+enum class publishing_mode_t {
+  ASYNCHRONOUS, // Asynchronous publishing mode
+  SYNCHRONOUS,  // Synchronous publishing mode
+  AUTO          // Use publishing mode set in XML file or Fast DDS default
```

Comment:
nitpick: This change seems unnecessary to me, maybe we should drop it?

Reply:
Removed whitespace changes in `custom_participant_info.hpp` in 65de1fb